src/interfaces/fees.ts

```diff
@@ -6,18 +6,26 @@ import { PROTOCOL_FEES } from "../services/subgraph/queries";
 import { FeesResponse } from "../services/subgraph/responses";
 import { Usdc } from "../types";
 
+const BigZero = new BigNumber(0);
+
 export class FeesInterface<C extends ChainId> extends ServiceInterface<C> {
   async protocolFees(since: Date): Promise<Usdc> {
     const response = await this.yearn.services.subgraph.fetchQuery<FeesResponse>(PROTOCOL_FEES, {
       sinceDate: since.getTime().toString(),
     });
 
+    let result = BigZero;
```

Comment:
I'm not sure about this. Firstly it should just be a `const` and secondly it hides what it is from line 20, it almost seems like it's a `result` but it's just a 0 we want to return. I'd personally remove this and just have below 

```typescript
return BigZero.toFixed(0);
```